documentation/assemblies/kafka-connect/assembly-kafka-connect.adoc

```diff
@@ -0,0 +1,34 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration.adoc
+
+[id='assembly-kafka-connect-{context}']
+= Kafka Connect/S2I cluster configuration
+
+This section describes how to configure a Kafka Connect or Kafka Connect with Source-to-Image (S2I) deployment in your Strimzi cluster.
+
+Kafka Connect is an integration toolkit for streaming data between Kafka brokers and other systems using `Connector` plugins.
+Kafka Connect provides a framework for integrating Kafka with an external data source or target, such as a database, for import or export of data using connectors.
+Connectors are plugins that provide the connection configuration needed.
+
+If you are using Kafka Connect, you configure either the `KafkaConnect` and `KafkaConnectS2I` resource.
+Use the `KafkaConnectS2I` resource if you are using the {docs-okd-s2i} framework to deploy Kafka Connect.
+
+* The full schema of the `KafkaConnect` resource is described in xref:type-KafkaConnect-reference[].
```

Comment:
Is it worth mentioning that the schema also includes examples?

Reply:
H'm. Maybe not required at this level of the doc. The procedure shows a full example and the links in the procedures go to the schema/examples, so maybe we're covered where we need to be.